src/Components/Typography/Typography.tsx

```diff
@@ -0,0 +1,12 @@
+import { TypographyProps } from "./types";
+import { StyledTypography } from "./styles";
+
+const Typography = ({ label, type, children }: TypographyProps) => {
+  return (
+    <div>
+      <StyledTypography type={type}>{children || label}</StyledTypography>
+    </div>
```

Comment:
can remove the wrapping `<div><div/>` and return directly the `<StyledTypography type={type}>{children || label}</StyledTypography>`

Reply:
done